package hsp.exception;

public class Exception01 {
    public static void main(String[] args) {
        int num1 = 10;
        int num2 = 0;
        //使用try-catch
        System.out.println(num1/num2);
    }


}
